n2t_hdl/src/channel_range.rs

```diff
@@ -1,39 +1,63 @@
 use ::std::ops::RangeInclusive;
 
+/// The range of a sub-bus of a non-internal bus (i.e., specified as IN or OUT)
+///
+/// Inclusive on `start` and `end`
+///
+/// `start` can't be greater than `end`
```

Comment:
I'd leave this out

Reply:
The `start` can't be greater than `end` part
other than that I think it's good

Reply:
when the